test/suite/test_chunkcache03.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python3
+#
+# Public Domain 2014-present MongoDB, Inc.
+# Public Domain 2008-2014 WiredTiger, Inc.
+#
+# This is free and unencumbered software released into the public domain.
+#
+# Anyone is free to copy, modify, publish, use, compile, sell, or
+# distribute this software, either in source code form or as a compiled
+# binary, for any purpose, commercial or non-commercial, and by any
+# means.
+#
+# In jurisdictions that recognize copyright laws, the author or authors
+# of this software dedicate any and all copyright interest in the
+# software to the public domain. We make this dedication for the benefit
+# of the public at large and to the detriment of our heirs and
+# successors. We intend this dedication to be an overt act of
+# relinquishment in perpetuity of all present and future rights to this
+# software under copyright law.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+# EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+# MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+# IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+
+import os, sys
+import random
+import threading
+import time
+import wiredtiger, wttest
+
+from random import randrange
+from wtdataset import SimpleDataSet
+from wtscenario import make_scenarios
+
+'''
+- Evaluating chunkcache performance both in-memory and on-disk, to test the functionality of pinned chunks.
```

Comment:
NP: In our test headers / comments we don't typically use continuous tense e.g evaluating -> evaluate.  Just a NP though, feel free to merge without change.

Reply:
Updated in [c2195b0](https://github.com/wiredtiger/wiredtiger/pull/9555/commits/c2195b03581ef2cd05b1229fde134027b54fc589)